tasks/ssh_deploy.js

```diff
@@ -15,6 +15,10 @@ var getScpOptions = function(options) {
         username: options.username
     };
 
+    if (options.passphrase) {
+        scpOptions.passphrase = options.passphrase;
+    }
+
     if (options.privateKey) {
         scpOptions.privateKey = options.privateKey;
```

Comment:
`passphrase` is only applicable when `privatekey` is used, IMO setting the option should go in this if block.